### PR TITLE
feat: add image component

### PR DESCRIPTION
**Pull request by sneh258**
Create Image Component
Handled edge cases (where image will be a circle)
Add test cases
Screenshot 2023-03-04 at 6 57 09 PM

### CODE REVIEW

File: src/components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.circle-image {
+  border-radius: 50%;
```
Comment (Chetan-MckinseyBEBR) [inline src/components/Image/image.css:8]: this class is not necessary. the border radius should be in the img style and it should inherit

File: src/components/Image/image.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+img {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/Image/image.css:1]: this will apply to all the img tags. instead have a unique class for ur functional component

File: src/components/Image/index.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './image.css';
+import PropTypes from 'prop-types';
+
+export default function Image({ imageUrl, altText, isACircle }) {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/Image/index.jsx:5]: No need for isACircle prop. Ur component css for image should inherit the border radius from parent div

**Review by Chetan-MckinseyBEBR — COMMENTED** [review]
Added few changes

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Review by rohini-senthil — APPROVED** [review]
